content/en/blog/2022/apisix.md

```diff
@@ -1,8 +1,9 @@
 ---
-title: Apache APISIX Integrates with OpenTelementry to Collect Tracing Data
+title: Apache APISIX Integrates with OpenTelemetry to Collect Tracing Data
 linkTitle: Apache APISIX-Opentelemetry Integration
 date: 2022-03-26
 author: Haochao Zhuang, Fei Han
+canonical_url: https://apisix.apache.org/blog/2022/02/28/apisix-integration-opentelemetry-plugin/
```

Comment:
I'm unsure what our policy will be about canonical URLs for regular blog posts (btw, this doesn't set the page's canonical URL yet). I'll bring this up during our next Comms/website meeting.

Reply:
Is it required to add `[original post]: {{% param canonical_url %}}` as well to complete the canonical URL?

Reply:
You can certainly add that to the end of your post, similarly to what was done for https://opentelemetry.io/blog/2022/troubleshooting-nodejs/ (see the end of the page). That is:

> _A version of this article was [originally posted](#) on the Apache APISIX blog._

Using `{{% param canonical_url %}}` for the link definition. See https://github.com/open-telemetry/opentelemetry.io/blob/main/content/en/blog/2022/troubleshooting-nodejs.md for an example.

Reply:
Ah I get it now, thanks @chalin!